File: ios/Runner/GATTProfile.swift
import Foundation
import CoreBluetooth

enum GATTProfile {
  static let serviceUUID = CBUUID(string: "9E2A0001-4B5A-4F5E-9A9D-1B7A00000001")
  static let reqCharUUID = CBUUID(string: "9E2A0002-4B5A-4F5E-9A9D-1B7A00000002")
  static let chunkCharUUID = CBUUID(string: "9E2A0003-4B5A-4F5E-9A9D-1B7A00000003")
  static let ackCharUUID = CBUUID(string: "9E2A0004-4B5A-4F5E-9A9D-1B7A00000004")

  static let magic: UInt16 = 0x6274 // "bt"
  static let maxTeaserUTF8Bytes: Int = 24
  static let maxCompactLocalNameUTF8Bytes: Int = 28
  static let previewBytes: Int = 120
}

File: android/app/src/main/kotlin/jp/cloxs/bittra/ble/GATTProfile.kt
package jp.cloxs.bitra.ble

import java.util.UUID

object GATTProfile {
    val SERVICE_UUID: UUID = UUID.fromString("9E2A0001-4B5A-4F5E-9A9D-1B7A00000001")
    val REQ_CHAR_UUID: UUID = UUID.fromString("9E2A0002-4B5A-4F5E-9A9D-1B7A00000002")
    val CHUNK_CHAR_UUID: UUID = UUID.fromString("9E2A0003-4B5A-4F5E-9A9D-1B7A00000003")
    val ACK_CHAR_UUID: UUID = UUID.fromString("9E2A0004-4B5A-4F5E-9A9D-1B7A00000004")

    const val MAGIC: Int = 0x6274 // "bt"
    const val MAX_TEASER_UTF8_BYTES = 20
    const val MANUFACTURER_ID = 0x0118
    const val PREVIEW_BYTES = 120
}
